### PR TITLE
Added attributes to other monsters

### PR DESCRIPTION
**Pull request by GReturn**

### CODE REVIEW

**Review by Nhalrath — CHANGES_REQUESTED** [review]
Wouldn't be needing those Potions. Just change them with null values

File: src/main/resources/assets/enemy.json [hunk 1]
```diff
@@ -80,20 +80,163 @@
...
+	"Spider": {
+		"NAME": "Spider",
+		"HP": 7,
+		"MP": 14,
+		"ATK": 15,
+		"DEF": 5,
+		"EXP": 9,
+		"MONEY": [2, 5, 7, 13],
+		"DROPS": {
+			"DROP_01": {"ID": "POTION", "AMOUNT": 1, "RATE": 47.0}
```
Comment (Nhalrath) [inline src/main/resources/assets/enemy.json:179]:
```suggestion
			"DROP_01": {"ID": "SPIDER_SILK", "AMOUNT": 1, "RATE": 47.0}
```

Comment (GReturn) [reply src/main/resources/assets/enemy.json]: Some item drops could be useless. Proceed anyway?

Comment (Nhalrath) [reply src/main/resources/assets/enemy.json]: Yep! Spider Silk can be used as a crafting material

Comment (Nhalrath) [reply src/main/resources/assets/enemy.json]: By Black Widow, it is a spider, right? If so, then it should have Spider Silk as a drop

Comment (GReturn) [reply src/main/resources/assets/enemy.json]: Do I just randomly add a drop item that relates to the monster?

Comment (Nhalrath) [reply src/main/resources/assets/enemy.json]: So long as the material can be realistically used as a crafting material

Comment (GReturn) [reply src/main/resources/assets/enemy.json]: Proceed with similar syntax?

` { "ID" : "POTION", "AMOUNT" : 1, "RATE" : 78.0 }`

Comment (GReturn) [reply src/main/resources/assets/enemy.json]: oh nvm

**Review by Nhalrath — APPROVED** [review]

**Pull request merged**